### PR TITLE
Add aria-label to header

### LINKED ISSUE #10008: Add alt text on navigation and header items
Much of the text on Habitica's website either has no alt text or is not allowing screen readers to read the alt text. This is a serious accessibility issue as it means that anything that is represented solely by a graphic is essentially invisible to blind users of the site. 

In some cases there is no alt text, but there is a tooltip. However, the screenreader will not read out the text unless you manage to get your cursor over the tooltip -- for which you have to know it's there.

The following screenshots illustrate just some of the areas I found on the site that are lacking alt text. Red boxes indicate areas "invisible" to the screenreader. Tested using NVDA, a free screenreader available online [here](https://www.nvaccess.org/). 

We should probably discuss what we want the alt text to read and what is most helpful/appropriate in various cases, if this is not simply a display issue. There is some advice on how to form useful alt text [here](https://webaim.org/techniques/alttext/).

screenshot one
screenshot two
screenshot three
screenshot four
screenshot five
screenshot six
screenshot seven

### PR DESCRIPTION
**Pull request by projectormato**
Fixes #10008

### Changes
- Add aria-label `classBadge.vue`

----
UUID: df9dbe38-5bb5-4d23-9ba5-b5e11ed03c86

### CODE REVIEW

**Comment by projectormato** [comment]
@paglias 
Sorry for the delay in work.
Added aria-label to the header class badge.

I have  two questions.
- `statsbar.vue` has tooltip, need aria-label?
- Are there any other elements that require aria-label?

**Comment by paglias** [comment]
Hi @projectormato and sorry for the delay!

- statsbar.vue: Looking at the accessibility section for bootstrap tooltips https://bootstrap-vue.org/docs/components/tooltip I'd say that it's not necessary as `aria` support is bundled in Bootstrap Vue
- @Tressley given that items with tooltips are already taken care of (see the point above) I'm not sure there's anything else that needs an `aria` label or `alt` tag for the first pass we wanted to do

**Comment by Tressley** [comment]
Okay, easy enough. I think this is good to go then.

**Comment by paglias** [comment]
actually @projectormato the logo in the top left (an svg image) is missing an `:aria-label`, can you add it? The `habitica` string should be ok (`$t('habitica')`)

**Comment by projectormato** [comment]
@paglias 
Thank you very much! 
OK, I'll do it now.

**Comment by projectormato** [comment]
@paglias 
Added `:aria-label` to the logo in the top left!
(I deleted the duplicate aria-label to the logo in the top left.)

File: website/client/src/components/header/menu.vue [hunk 1]
```diff
@@ -11,15 +11,16 @@
       type="dark"
       :class="navbarZIndexClass"
     >
-      <b-navbar-brand
```
Comment (paglias) [inline website/client/src/components/header/menu.vue]: @projectormato sorry I had missed the existing label here, could you revert your last commit?

**Review by paglias — COMMENTED** [review]
I've left one comment but it's looking good and I'll merge the pr as soon as that's handled

**Comment by projectormato** [comment]
@paglias 
Thank you for review!
Reverted my last commit.
https://github.com/HabitRPG/habitica/pull/12584/commits/8ce34fc3a9e6e9a19fbf2653b9ef5b7dd0176165

**Comment by paglias** [comment]
@projectormato  thanks for this! I'm merging the PR and awarding your contributor tier.

Note that further tiers require increasing amounts of work from one to the next, so Tier 2 may require a couple of PRs or a larger PR to attain. But keep helping out and we'll express our gratitude accordingly!

**Pull request merged**